scripts/microbe_stage/microbe_operations.as

```diff
@@ -859,11 +860,12 @@ ObjectID spawnMicrobe(CellStageWorld@ world, Float3 pos, const string &in specie
         // the physics size matches the rendered size
         node.Scale = Float3(0.5, 0.5, 0.5);
         node.Marked = true;
+
         // This call is also not the cheapest. So would be much better
         // if the physics generation actually did the right then when
         // species.isBacteria is true
-        physics.ChangeShape(world.GetPhysicalWorld(),
-            world.GetPhysicalWorld().CreateSphere(HEX_SIZE/2.0f));
+        rigidBodyComponent.ChangeShape(world.GetPhysicalWorld(),
+            rigidBodyComponent.Body.Shape);
```

Comment:
Doesn't this result in the bacteria not having a smaller physics size?

Reply:
By testing i find it quite precise, should be not fitting good?

Reply:
I don't remember there being checks for being a bacteria in the physics shape creation in the individual organelles. That means that with this change (which should btw do nothing as the shape is already applied) all bacteria have physics size be the same as eukaryotes.

Reply:
Once the proper size is handled elsewhere this whole change shape call should be removed. Also there needs to be a comment in the restore organelle layout that if the "isBacteria" changes all organelles *must* be readded (to make sure no one introduces bugs in the future).